src/pages/index.js

```diff
@@ -123,122 +112,62 @@ export default function IndexPage({ pkgVersion }) {
 
       <IconModal icon={iconInView} onDismiss={() => setIconInView(null)} />
 
-      <header
-        className="px-4 sm:px-6 lg:px-16 bg-gray-800 pb-32 bg-cover bg-no-repeat bg-center"
-        style={{
-          backgroundImage: `url(${process.env.BACKEND_URL}/bg-cover.svg)`,
-        }}
-      >
-        <div className="max-w-limit mx-auto flex justify-between mb-16 ">
-          <div className="py-12 flex items-center text-white space-x-2">
-            <Chromicons />
+      <Header pkgVersion={pkgVersion} />
 
-            <dl className="inline-flex self-end items-center px-2 rounded-full text-xs bg-gradient-to-r from-super-orange to-super-blue">
-              <dt className="sr-only">Chromicons version</dt>
-              <dd className="text-black">{pkgVersion}</dd>
+      <main className="text-gray-600 flex flex-1 flex-col scrolling-touch w-3/4 m-auto bg-transparent">
+        <div className="flex sm:flex-col xl:flex-row relative mb-16">
+          <div>
+            <h1 className="text-black text-2xl leading-9 font-bold sm:text-4xl sm:leading-10 uppercase">
+              Handcrafted open source icons
+            </h1>
+            <p className="text-black max-w-lg text-sm sm:text-base pt-5 pb-5">
+              Ready to use in web, iOS, Android, and desktop apps. Support for
+              SVG and web font. Completely open source, MIT licensed, and built
+              with
+              <span className="text-red-600" role="img">
```

Comment:
I missed this first time around, but we should add the following for accessibility reasons:

```suggestion
              <span className="text-red-600" role="img" aria-label="love">
```

https://accessibilityinsights.io/info-examples/web/role-img-alt/